docs/framework/data/wcf/accessing-data-service-resources-wcf-data-services.md

```diff
@@ -16,43 +16,43 @@ ms.assetid: 9665ff5b-3e3a-495d-bf83-d531d5d060ed
  In [!INCLUDE[ssODataShort](../../../../includes/ssodatashort-md.md)], you address any data exposed by the data model by using a URI. For example, the following URI returns a feed that is the Customers entity set, which contains entries for all instances of the Customer entity type:  
   
 ```  
-http://services.odata.org/Northwind/Northwind.svc/Customers  
+https://services.odata.org/Northwind/Northwind.svc/Customers  
```

Comment:
not sure whether this can be updated or not. I'd roll back the change for URIs too